langchain4j/src/main/java/dev/langchain4j/memory/chat/ChatMemories.java

```diff
@@ -0,0 +1,40 @@
+package dev.langchain4j.memory.chat;
+
+import dev.langchain4j.memory.ChatMemory;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.WeakHashMap;
+
+public class ChatMemories {
```

Comment:
I encapsulated the chat memory related concern into this class, in a similar way as I did for the `ToolService` one introduced when I refactored the tools management.

Reply:
Ok, I will rename this, and yes I expect that this should be used by the quarkus extension.